packages/jwt/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cfworker/jwt",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
hey thanks for the PR! Could you revert this change? I'll make the version bump with the automated changeset process.

Reply:
Done.